### ISSUE TITLE
How can I avoid triggering the security verification of web pages when using playwright?

### ISSUE THREAD

**Issue by jiangyijin** [microsoft/playwright#15643]
How can I avoid triggering the security verification of web pages when using playwright? I'm a novice and have a passion for playwright, but I'm always blocked by the security verification of web pages. What can I do to solve this problem?

My initial idea is to connect to a manually opened browser, but I tried many methods and failed. Please give me some advice. Thank you very much.

**Comment by yury-s** [comment]
Closing this issue as I assume the reaction to the previous comment and lack of follow-up messages means that the question was answered. If you have further questions feel free to file a new issue and reference this one.
